apps/src/applab/applab.js

```diff
@@ -1246,6 +1255,54 @@ Applab.onDivApplabClick = function (event) {
   Applab.editElementProperties(event.target);
 };
 
+/**
+ * @param el {Element}
+ * @returns {number} The outerWidth (width + margin) of the element in pixels,
+ * or NaN if element's css width or margin are not defined.
+ */
+Applab.getOuterWidth = function(el) {
+  var marginLeft = parseInt($(el).css('margin-left'));
+  var marginRight = parseInt($(el).css('margin-right'));
```

Comment:
I guess we don't have this jshint option turned on, but it sounds like you can get some unexpected results if you don't specify a radix:
https://jslinterrors.com/missing-radix-parameter